content/en/blog/_posts/2024-08-21-cgroup-driver-from-cri.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: blog
+title: "Configuring Kubernetes just got easier: cgroup driver from CRI"
+date: 2024-07-01
+slug: cgroup driver from CRI to betra
+author: >
+  Peter Hunt (Red Hat)
+---
+
+Historically, a pain point for users running new Kubernetes clusters has been
```

Comment:
Converting to active voice, per [style guide](https://kubernetes.io/docs/contribute/style/style-guide/#use-active-voice). 

Historically, the cgroup driver has been a pain point for users running new Kubernetes clusters.